docs/deployment/deploy-to-containarium.md

```
@@ -0,0 +1,284 @@
+  "service": "Insforge OSS Backend",
+  "timestamp": "..."
+}
+```
+
+### 6. Connect your agent to InsForge MCP
+
+Open `https://<your-subdomain>` in a browser and follow the in-product flow to connect your MCP-compatible agent (Cursor, Claude Code, Windsurf, OpenCode, etc.) to the InsForge MCP server.
+
+Verify the connection by sending this prompt to your agent:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)